lib/ansible/plugins/connection/docker.py

```diff
@@ -50,6 +50,7 @@ class Connection(ConnectionBase):
 
     transport = 'docker'
     has_pipelining = True
+    remote_user = None
```

Comment:
We wan this to be an instance attribute instead of a class attribute.  A class attribute will only have one value that's shared among all classes.